funcx/executors/high_throughput/worker_map.py

```diff
@@ -42,11 +45,10 @@ def scrub_worker(self, worker_id):
 
         worker_type = self.worker_types[worker_id]
 
-        logger.debug("In KILL worker worker_id: {} type: {}".format(worker_id, worker_type))
+        # logger.debug("In KILL worker worker_id: {} type: {}".format(worker_id, worker_type))
+        assert(self.worker_counts[worker_type] >= 1)
```

Comment:
I am not sure. But is try catch better than `assert` here?

Reply:
Discussed with ZZ/Ryan. Plan is to take these out, and then later (0.2.0) build a testing suite for the schedulers. Was mainly here to ensure we weren't killing containers that didn't exist OR to not spin up a negative number of containers. (i.e., the scheduler would be broken).